x-pack/test/functional_endpoint/apps/endpoint/resolver.ts

```diff
@@ -0,0 +1,281 @@
+          parseFloat(OriginalNodeDataStyle[i].left)
+        );
+        expect(parseFloat(CenterNodeDataStyle[i].top)).to.equal(
+          parseFloat(OriginalNodeDataStyle[i].top)
+        );
+      }
+    });
+
+    it('resolver Nodes navigation zoom in', async () => {
+      const OriginalNodeData = await pageObjects.endpointAlerts.getEndpointAlertResolverNodeData(
+        'resolverNode',
+        'style'
+      );
+      const OriginalNodeDataStyle = await pageObjects.endpointAlerts.parseStyles(OriginalNodeData);
+
+      await (await testSubjects.find('zoom-in')).click();
+      await browser.setWindowSize(2400, 1800);
+      const NewNodeData = await pageObjects.endpointAlerts.getEndpointAlertResolverNodeData(
+        'resolverNode',
+        'style'
+      );
+      const NewNodeDataStyle = await pageObjects.endpointAlerts.parseStyles(NewNodeData);
+
+      for (let i = 1; i < NewNodeDataStyle.length; i++) {
+        expect(parseFloat(OriginalNodeDataStyle[i].left)).to.lessThan(
+          parseFloat(NewNodeDataStyle[i].left)
+        );
+        expect(parseFloat(OriginalNodeDataStyle[i].top)).to.lessThan(
+          parseFloat(NewNodeDataStyle[i].top)
+        );
+        expect(parseFloat(OriginalNodeDataStyle[i].width)).to.lessThan(
+          parseFloat(NewNodeDataStyle[i].width)
+        );
+        expect(parseFloat(OriginalNodeDataStyle[i].height)).to.lessThan(
+          parseFloat(NewNodeDataStyle[i].height)
+        );
+        await (await testSubjects.find('zoom-out')).click();
+      }
+    });
+
+    it('resolver Nodes navigation zoom out', async () => {
+      const OriginalNodeData = await pageObjects.endpointAlerts.getEndpointAlertResolverNodeData(
+        'resolverNode',
+        'style'
+      );
+      const OriginalNodeDataStyle = await pageObjects.endpointAlerts.parseStyles(OriginalNodeData);
+      await (await testSubjects.find('zoom-out')).click();
+      const NewNodeData = await pageObjects.endpointAlerts.getEndpointAlertResolverNodeData(
+        'resolverNode',
+        'style'
+      );
+      const NewNodeDataStyle1 = await pageObjects.endpointAlerts.parseStyles(NewNodeData);
+
+      for (let i = 1; i < OriginalNodeDataStyle.length; i++) {
+        expect(parseFloat(NewNodeDataStyle1[i].left)).to.lessThan(
+          parseFloat(OriginalNodeDataStyle[i].left)
+        );
+        expect(parseFloat(NewNodeDataStyle1[i].top)).to.lessThan(
+          parseFloat(OriginalNodeDataStyle[i].top)
+        );
+        expect(parseFloat(NewNodeDataStyle1[i].width)).to.lessThan(
+          parseFloat(OriginalNodeDataStyle[i].width)
+        );
+        expect(parseFloat(NewNodeDataStyle1[i].height)).to.lessThan(
+          parseFloat(OriginalNodeDataStyle[i].height)
+        );
+      }
+      await (await testSubjects.find('zoom-in')).click();
+    });
+
+    after(async () => {
+      await (await testSubjects.find('euiFlyoutCloseButton')).click();
```

Comment:
just out of curiosity, I know a lot of the devs were putting in pretty good unit tests for their UI components to validate they are doing basic things like hiding / showing... do we know that this indeed hides the Flyout because of a different test?   If not, we might put in a validation that it isn't rendered anymore while we're at this point in the code.